stride/problem/data.py

```diff
@@ -1562,13 +1503,47 @@ def plot_one(self, id, **kwargs):
 
         return axis
 
-    def _resample(self, factor, new_num, **kwargs):
+    def _resample(self, old_step, new_step, new_num, freq_niquist, len_hann=None, mute_start=None, **kwargs):
+        '''
+        In-place operation to resample a trace to a new time-spacing.
+        Sinc interpolation is used.
+
+        Parameters
+        ----------
+        factor
+            The ratio dt_old/dt_new.
+        new_num
+            The length of the trace. 
+        freq_niquist
+            min(fs_old, fs_new), where fs is the sampling frequency.
+        len_hann
+            The length of the Hann window applied around the edge of the mute.
+        mute_start, default: None
+            The length of the starting mute in num_samples.
+        filter_type_antialias, default: cos
```

Comment:
this should be `filter_type_antialias : str, optional` and the line below should have the default

Reply:
done